examples/textual_inversion/textual_inversion_mlflow.py

```diff
@@ -907,8 +901,9 @@ def main():
                 noise = torch.randn_like(latents)
                 bsz = latents.shape[0]
                 # Sample a random timestep for each image
-                timesteps = torch.randint(0, noise_scheduler.config.num_train_timesteps, (bsz,), device=latents.device)
-                timesteps = timesteps.long()
+                torch.manual_seed(42)
```

Comment:
I also prefer keeping --seed option as Loc said.

Reply:
@yeomni I just replace `42` by `arg.seed` in the latest PR